Frontend/CO.CDP.OrganisationApp/ThirdPartyApiClients/CompaniesHouse/CompaniesHouse.cs

```diff
@@ -0,0 +1,70 @@
+using Flurl;
+using Flurl.Http;
+
+namespace CO.CDP.OrganisationApp.ThirdPartyApiClients.CompaniesHouse;
+
+public class CompaniesHouseApi(IConfiguration configuration, ILogger<CompaniesHouseApi> logger) : ICompaniesHouseApi
+{
+    public async Task<RegisteredAddress?> GetRegisteredAddress(
+        string companyNumber)
+    {
+        var companiesHouseUrl = GetComapniesHouseUrl();
```

Comment:
Typo - `Comapnies` should be `Companies`

Reply:
Updated.